src/models/broadcastZone.ts

```diff
@@ -57,7 +60,7 @@ export class BroadcastZone
     }
   }
 
-  public tryUnplace(userID: string) {
+  public tryUnplace(userID: string, _: number = -1) {
```

Comment:
That's because the interface definition requires it, since some zones also have relevant spots (like the desk zone) to use during placement. The broadcast zone only has one spot so the spot ID isn't relevant in the implementation itself, but to satisfy the interface it still needs to allow for that argument.

Reply:
gotcha, thanks!